ingest-app/src/cmr/ingest/site/routes.clj

```diff
@@ -0,0 +1,47 @@
+(ns cmr.ingest.site.routes
+  "Defines the HTTP URL routes for the ingest web site."
+  (:require
+   [cmr.common-app.static :as static]
+   [cmr.common-app.site.data :as common-data]
+   [cmr.common-app.site.pages :as common-pages]
+   [cmr.ingest.site.pages :as pages]
+   [cmr.transmit.config :as config]
+   [compojure.core :refer [GET context routes]]
+   [ring.util.response :refer [redirect]]))
+
+(defn build-routes [system]
+  (routes
+    (context (get-in system [:ingest-public-conf :relative-root-url]) []
+      (GET "/"
+           {ctx :request-context}
+           (pages/home ctx))
```

Comment:
We could do, no problem. All three projects will need to be updated to do this, and it will need some testing, since the routes will need to be updated and some other bits will change a little bit.

I'll open a ticket for that.